### ISSUE TITLE
RCS1213 (UnusedMember) does not handle recursion calls

### ISSUE THREAD

**Issue by JosefPihrt** [JosefPihrt/Roslynator#308]
```csharp
private void UnusedMethod() // not reported as unused method
{
  UnusedMethod();
}
```
